monorepo-artifacts-url

```diff
@@ -0,0 +1 @@
+https://storage.googleapis.com/oplabs-contract-artifacts/artifacts-v1-3e3960bdce30602d0d92a76cb2d50efa4f1f4befced161d2186ad10134c8c077.tar.gz
```

Comment:
it could be cool once we had an admin rpc in this to also output the artifact checksum it's using